templates/bridgeserver2.yaml

```diff
@@ -410,6 +410,9 @@ Resources:
         - Namespace: 'aws:elasticbeanstalk:application:environment'
           OptionName: cuimc.prod.password
           Value: !Ref CuimcProdPassword
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: participant-file.bucket
```

Comment:
I matched this `OptionName` with the new entry `participant-file.bucket` in `BridgeServer2.conf` for the participant file service.

Reply:
I don't see `participant-file.bucket` anywhere in this repo.  how was that bucket created?

Reply:
This is an environment var added by BridgeServer.